imgur.sh

```diff
@@ -89,14 +89,17 @@ while [ $# -gt 0 ]; do
 	url="${url%%</link>*}"
 	delete_hash="${response##*<deletehash>}"
 	delete_hash="${delete_hash%%</deletehash>*}"
-	echo $url
-	echo "Delete page: https://imgur.com/delete/$delete_hash" >&2
+	delete_page="https://imgur.com/delete/$delete_hash"
+	echo $url | sed 's/^http:/https:/'
```

Comment:
I only set $delete_page because it's used again below in logging; I do this for every variable used more than once. The logging echo could perfectly use "https://imgur.com/delete/$delete_hash" instead. Opinion?

Reply:
No strong opinion. The only important thing to me here was to not let sed creep back in, which would be a regression. Then I was dictating the order of lines but I don't feel that strongly about it. I wasn't complaining about making an extra variable -- I see why you did that and it makes sense.